add duration (#984)
vesoft-inc/nebula-docs: docs-2.0/3.ngql-guide/3.data-types/4.date-and-time.md

# Date and time types

This topic will describe the `DATE`, `TIME`, `DATETIME`, `TIMESTAMP`, and `DURATION` types.

## Precautions

- While inserting time-type property values with `DATE`, `TIME`, and `DATETIME`, Nebula Graph transforms them to a UTC time according to the time zone specified with the `timezone_name` parameter in the [configuration files](../../5.configurations-and-logs/1.configurations/1.configurations.md). The time-type values returned by nGQL queries are all UTC time.

  !!! note

>         To change the time zone, modify the `timezone_name` value in the configuration files of all Nebula Graph services.

Reviewer: ```suggestion
        To change the timezone, modify the `timezone_name` value in the configuration files of all Nebula Graph services.
```

Reviewer: `timezone` is the favored spelling in the US

> - `date()`, `time()`, and `datetime()` can transform a time-type property with a specified time zone. For example, `datetime("2017-03-04 22:30:40.003000+08:00")` or `datetime("2017-03-04T22:30:40.003000[Asia/Shanghai]")`.

Reviewer 2: ```suggestion
- `date()`, `time()`, and `datetime()` can convert a time-type property with a specified time zone. For example, `datetime("2017-03-04 22:30:40.003000+08:00")` or `datetime("2017-03-04T22:30:40.003000[Asia/Shanghai]")`.
```

Reviewer 2: it's better to use convert instead of transform, transform would mean the size, shape, color, or some other attribute was changed.
just one question: what will the xxx convert to or from?

> - `date()`, `time()`, and `datetime()` can transform a time-type property with a specified time zone. For example, `datetime("2017-03-04 22:30:40.003000+08:00")` or `datetime("2017-03-04T22:30:40.003000[Asia/Shanghai]")`.

Reviewer: ```suggestion
- `date()`, `time()`, and `datetime()` can transform a time-type property with a specified timezone. For example, `datetime("2017-03-04 22:30:40.003000+08:00")` or `datetime("2017-03-04T22:30:40.003000[Asia/Shanghai]")`.
```

- The underlying storage data type is **int64**.

## DURATION

> The `DURATION` data type is used for indicates a period of time. Map data that are freely combined by `years`, `months`, `days`, `hours`, `minutes`, and `seconds` indicates the `DURATION`.

Reviewer: ```suggestion
The `DURATION` data type is used to indicate a period of time. Map data that are freely combined by `years`, `months`, `days`, `hours`, `minutes`, and `seconds` indicates the `DURATION`.
```

Reviewer: why the example in Chinese docs is missing?
